### PR TITLE
Add a few useful methods to Direction

### PR DESCRIPTION
**Pull request by Minikloon**

### CODE REVIEW

File: src/main/java/net/minestom/server/utils/Direction.java [hunk 1]
```diff
@@ -15,11 +16,13 @@ public enum Direction {
     private final int normalX;
     private final int normalY;
     private final int normalZ;
+    private final Vec normalVec;
```
Comment (MelonHell) [inline src/main/java/net/minestom/server/utils/Direction.java:19]: I think we should remove normalVec

Comment (mworzala) [reply src/main/java/net/minestom/server/utils/Direction.java:19]: Why? it seems ok to me

Comment (MelonHell) [reply src/main/java/net/minestom/server/utils/Direction.java:19]: It just seems weird to me to store the same data twice

Comment (mworzala) [reply src/main/java/net/minestom/server/utils/Direction.java:19]: There are 6 of these ever instantiated. It's negligible and avoids a lot of allocation

**Pull request merged**